torch/_inductor/codegen/cpp_wrapper_gpu.py

```diff
@@ -571,26 +571,31 @@ def generate_kernel_call(
                 device_index, call_args
             )
             kernel_var_name = self.generate_load_kernel_once(kernel_name, V.graph)
+            arg_signatures = []
             if triton_version_uses_attrs_dict():
-                signature = triton_meta["signature"]
-                arg_signatures = [
-                    val for val in signature.values() if val != "constexpr"
-                ]
-                call_args = [
-                    call_arg
-                    for call_arg, arg_name in zip(call_args, signature)
-                    if signature[arg_name] != "constexpr"
-                ]
-                arg_types = [
-                    arg_type
-                    for arg_type, arg_name in zip(arg_types, signature)
-                    if signature[arg_name] != "constexpr"
-                ]
+                if (
+                    triton_meta is not None
+                    and triton_meta.get("configs")
+                    and triton_meta.get("signature")
+                ):
```

Comment:
minor nit: could we put this check on the outside (if triton_meta is not None...) and put the `triton_version_uses_attrs_dict` check inside that?

Reply:
good nit!